src/software/thunderscope/binary_context_managers/BUILD

```diff
@@ -0,0 +1,13 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@thunderscope_deps//:requirements.bzl", "requirement")
+
+py_library(
+    name = "binary_context_managers",
+    srcs = [
+        "full_system.py",
+        "game_controller.py",
+        "simulator.py",
+        "util.py",
+    ],
+)
```

Comment:
I am not sure what you want. Do you want me to move it to another folder?

From my understanding, both thunderscope and simulated test seems to directly import Gamecontrollers and other things.

Reply:
You can keep it in the same folder, I'm suggesting that we divide this `py_library` into multiple. In addition to that, since some of these files/classes launch another process by running a compiled binary, it'd a good idea to have those binaries as dependencies of the new py libraries. Currently, the executables are in the [`py_library` for thunderscope](https://github.com/UBC-Thunderbots/Software/blob/83d226fe1e56d0ab93991a0acdae88a344ce0ec5/src/software/thunderscope/BUILD#L22) and simulated test fixture, but it would make more sense to be a dependency of the files here. 

Something like

```python
py_library(
    name = "full_system",
    srcs = [
        "full_system.py"
    ],
    data = [
        "//software:unix_full_system",
    ],
)


py_library(
    name = "simulator",
    srcs = [
        "simulator.py"
    ],
    data = [
        "//software:er_force_simulator_main",
    ],
)
```

And then remove these `data` dependencies from other `py_libraries`.

For game controller, we download the pre-built binary of it, so that wouldn't need the `data` field.

Reply:
Following up on this

Reply:
resolved